src/config/mongoose.js

```diff
@@ -8,7 +25,5 @@ module.exports = () => {
     require('../models/location.server.model');
     require('../models/service.server.model');
 
-    return {
-        connect: mongoose.connect(config('DB_URL'))
-    };
+    return mongoose.connect(config('DB_URL'));
```

Comment:
Just returning the connection instead of returning an object. At some point I was returning a logger instance inside this object but now there's a separate logging configuration, since it's not related to mongoose.